src/components/Dashboard/StudentData/index.js

```diff
@@ -60,7 +60,9 @@ function StudentData() {
     }).then((res) => {
       if (
         id == user.data.user.partner_id ||
-        user.data.user.rolesList.indexOf("admin") > -1
+        user.data.user.rolesList.indexOf("admin") > -1 ||
+        user.data.user.rolesList.indexOf("partner_view") > -1 ||
+        user.data.user.rolesList.indexOf("partner_edit") > -1
```

Comment:
Since the logic of checking that one Array contains at least one item from another Array appears multiple times, it may be good to add a function to [utils.js](https://github.com/navgurukul/bhanwari-devi/blob/main/src/common/utils.js) as follows:
```
const hasOneFrom = (arr1, arr2) => arr2.some((item) => arr1.includes(item));
```
Then you could write the following here (after exporting the function from utils.js and importing it from this file):
```suggestion
        hasOneFrom(user.data.user.rolesList, ["admin", "partner_view", "partner_edit"]);
```